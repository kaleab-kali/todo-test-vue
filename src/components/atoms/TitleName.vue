<template>
  <header>
    <h1 :class="$style.title">To do list</h1>
  </header>
</template>

<script>
export default {};
</script>

<style lang="scss" module>
.title {
  font-family: "Open Sans";
  background-color: $lightOrange;
  color: $brown;
  padding: 0.688rem 14.1rem;
  font-weight: 600;
  font-size: 1.25rem;
  text-align: center;
  @media (max-width: 980px) {
    padding: 0.688rem 10rem;
    font-size: 1.2rem;
  }
  @media (max-width: 850px) {
    padding: 0.688rem 5rem;
  }
  @media (max-width: 600px) {
    padding: 0.688rem 4rem;
  }
  @media (max-width: 450px) {
    padding: 0.688rem 2rem;
  }
}
</style>
