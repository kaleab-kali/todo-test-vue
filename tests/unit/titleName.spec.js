import { mount } from "@vue/test-utils";
import TitleName from "@/components/atoms/TitleName.vue";

describe("Atom TitleName Component", () => {
  it("Correct layout", () => {
    const wrapper = mount(TitleName);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
