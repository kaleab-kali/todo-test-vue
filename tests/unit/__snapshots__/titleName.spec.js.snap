// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom TitleName Component Correct layout 1`] = `<header><h1 class="title">to do list</h1></header>`;

exports[`Atom TitleName Component has name 1`] = `<header><h1 class="title">to do list</h1></header>`;

exports[`TitleName.vue has name 1`] = `<header><h1 class="title">to do list</h1></header>`;
